.storybook/stories/datagrid/datagrid-row.stories.ts

```diff
@@ -58,12 +58,18 @@ const RowTemplate: StoryFn = args => ({
         (clrDgExpandedChange)="index === 0 && clrDgExpandedChange($event)"
         (clrDgSelectedChange)="index === 0 && clrDgSelectedChange($event)"
       >
-        <clr-dg-cell>{{ element.name }}</clr-dg-cell>
-        <clr-dg-cell>{{ element.symbol }}</clr-dg-cell>
-        <clr-dg-cell>{{ element.number }}</clr-dg-cell>
+        <clr-dg-cell>{{ !(emptyRow && index === 0) ? element.name : '' }}</clr-dg-cell>
+        <clr-dg-cell>{{ !(emptyRow && index === 0) ? element.symbol : '' }}</clr-dg-cell>
+        <clr-dg-cell>{{ !(emptyRow && index === 0) ? element.number : '' }}</clr-dg-cell>
         <clr-dg-cell class="electronegativity-container">
-          {{ element.electronegativity }}
-          <div [style.width.%]="(element.electronegativity * 100) / 5" class="electronegativity-bar">&nbsp;</div>
+          {{ !(emptyRow && index === 0) ? element.electronegativity : '' }}
+          <div
+            *ngIf="!(emptyRow && index === 0)"
```

Comment:
```suggestion
            *ngIf="!emptyRow || index !== 0)"
```

I think this would be easier to read/understand. The unary not operator can get lost when reading an compound expression.

Reply:
Corrected in commit 09e8735